### PR TITLE
Dropdown submenu for the project-site

### PR DESCRIPTION
**Pull request by Vector-Blanxart**
Signed-off-by: Esteban Villegas Blanxart <evillegas@teclib.com>

### Changes description

Copy the changes made in SCSS and JavaScript of the glpi-plugin project to web-mdm-dashboard project.

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@Vector-Blanxart |20|

### References

Closes #N/A
Related #1042
Depends on #N/A

### CODE REVIEW

**Comment by ajsb85** [comment]
Hi, @Vector-Blanxart 
- The task related to update the gems is a commit message type `build`
- The task related to the code rules is a commit message type `style`

Hi, @Gianfranco97 
Could you provide your config environment to have the same code rules in the code editor of Villegas?

**Comment by Gianfranco97** [comment]
JS Rules:
- https://github.com/flyve-mdm/web-mdm-dashboard/blob/gh-pages/.eslintrc.json

Plugin of VS-Code:
- https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

File: .eslintrc.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "window": false
     },
     "rules": {
-        "semi": ["error", "never"]
+        "semi": ["error", "never"],
+        "linebreak-style": 0
```
Comment (Gianfranco97) [inline .eslintrc.json]: why did you disable this rule?

**Comment by Gianfranco97** [comment]
@Vector-Blanxart Could you add a test submenu and show us what it looks like?

File: .eslintrc.json (hunk 1 above)
Comment (Vector-Blanxart) [reply .eslintrc.json]: Because ESLint marks errors on all the lines.

**Comment by Gianfranco97** [comment]
And please resolve the conflicts with the SCSS file

File: .eslintrc.json (hunk 1 above)
Comment (Gianfranco97) [reply .eslintrc.json]: All the lines?...

Comment (ajsb85) [reply .eslintrc.json]: https://eslint.org/docs/rules/linebreak-style

Comment (Gianfranco97) [reply .eslintrc.json]: @Vector-Blanxart  The 'linebreak-style' rule works perfectly. check the configuration of your local.

pxngg-wdc9s

**Comment by Gianfranco97** [comment]
@Vector-Blanxart The submenus are not working properly. Please check that.

p5off-srnom

**Comment by Gianfranco97** [comment]
@Vector-Blanxart Just use the environment 'docs', please.

captura de pantalla 2018-08-17 a la s 11 14 43 a m

**Comment by Gianfranco97** [comment]
And do not repeat the same commit, make a 'squash' of the identical commits.

**Pull request merged**